src/index.js

```diff
@@ -25,6 +25,11 @@ export const or = (...rules) => ({
   rules,
 });
 
+export const xor = (...rules) => ({
```

Comment:
We will probably want a test or two making sure that this function exists, and that checks it's param signature. So something like 

```
assert.equal(typeof xor, 'function')
```